src/plugins/intel_gpu/include/intel_gpu/primitives/prior_box.hpp

```diff
@@ -135,10 +209,82 @@ struct prior_box : public primitive_base<prior_box> {
           clustered(true) {
     }
 
+    /// @brief Constructs prior-box primitive for dynamic shape, which executes clustered version.
+    prior_box(const primitive_id& id,
+              const std::vector<input_info>& inputs,
+              const bool clip,
+              const std::vector<float>& variance,
+              const float step_width,
+              const float step_height,
+              const float offset,
+              const std::vector<float>& widths,
+              const std::vector<float>& heights,
+              data_types output_dt,
+              const padding& output_padding = padding())
+        : primitive_base(id, inputs, {output_padding}, {optional_data_type{output_dt}}),
+          flip(false),
+          clip(clip),
+          variance(variance),
+          step_width(step_width),
+          step_height(step_height),
+          offset(offset),
+          scale_all_sizes(false),
+          widths(widths),
+          heights(heights),
+          clustered(true) {
+    }
+
+    PriorBoxV0Op::Attributes get_attrs_v0() const {
+        PriorBoxV0Op::Attributes attrs;
+        attrs.min_size = min_sizes;
+        attrs.max_size = max_sizes;
+        attrs.aspect_ratio = aspect_ratios;
+        attrs.density = density;
+        attrs.fixed_ratio = fixed_ratio;
+        attrs.fixed_size = fixed_size;
+        attrs.clip = clip;
+        attrs.flip = flip;
+        attrs.step = step;
+        attrs.offset = offset;
+        attrs.variance = variance;
+        attrs.scale_all_sizes = scale_all_sizes;
+        return attrs;
+    }
+
+    PriorBoxV8Op::Attributes get_attrs_v8() const {
+        PriorBoxV8Op::Attributes attrs;
+        attrs.min_size = min_sizes;
+        attrs.max_size = max_sizes;
+        attrs.aspect_ratio = aspect_ratios;
+        attrs.density = density;
+        attrs.fixed_ratio = fixed_ratio;
+        attrs.fixed_size = fixed_size;
+        attrs.clip = clip;
+        attrs.flip = flip;
+        attrs.step = step;
+        attrs.offset = offset;
+        attrs.variance = variance;
+        attrs.scale_all_sizes = scale_all_sizes;
+        return attrs;
+    }
+
+    PriorBoxClusteredOp::Attributes get_attrs_clustered() const {
+        PriorBoxClusteredOp::Attributes attrs;
+        attrs.widths = widths;
+        attrs.heights = heights;
+        attrs.clip = clip;
+        attrs.step_widths = step_width;
+        attrs.step_heights = step_height;
+        attrs.step = step;
+        attrs.offset = offset;
+        attrs.variances = variance;
+        return attrs;
+    }
+
     /// @brief Spatial size of generated grid with boxes.
-    tensor output_size{};
+    mutable cldnn::tensor output_size{};
```

Comment:
why should it be mutable? We dont change primitive's variables after creation.

Reply:
Removed 'mutable'. Thanks!